config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,12 @@
+CarrierWave.configure do |config|
+  require 'carrierwave/orm/activerecord'
+
+  config.fog_provider = 'fog/aws'                        # required
+  config.fog_credentials = {
+    provider:              'AWS',                        # required
+    aws_access_key_id:     ENV['AWS_ACCES_KEY_ID'],      # required
```

Comment:
better to use **ENV.fetch** because it raise an error if no keys found

Reply:
done